UWP/PPSSPP_UWPMain.cpp

```diff
@@ -528,7 +614,13 @@ bool System_MakeRequest(SystemRequestType type, int requestId, const std::string
 }
 
 void System_ShowFileInFolder(const char *path) {
-	// Unsupported
+	auto ver = Windows::System::Profile::AnalyticsInfo::VersionInfo;
+	if (ver->DeviceFamily == "Windows.Xbox") {
+		// Unsupported
+	}
+	else {
+		OpenFolder(std::string(path));
```

Comment:
Technically, the provided path is to a file, and we should be opening its parent folder.  On non-UWP this prompts Explorer to open its parent folder and to highlight the specified file.  Is `OpenFolder()` going to be happy with a filename?

-[Unknown]

Reply:
Oh, then this need to be solved within `OpenFolder`